Dockerfile.dapper

```diff
@@ -7,7 +7,9 @@ ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} DAPPER_ENV=REPO DAPPE
     OPERATOR_SDK_VERSION=0.12.0 GOROOT=/usr/lib/golang
 
 RUN curl -Lo /usr/bin/operator-sdk "https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu" && \
-    chmod a+x /usr/bin/operator-sdk
+    chmod a+x /usr/bin/operator-sdk && \
+    curl -LO "https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz" && \
```

Comment:
Could you make the version a variable, as in `Dockerfile.dapper-base`?

Reply:
Done.